docs/undergrad.yaml

```diff
@@ -587,7 +587,6 @@ Distribution calculus:
     Dirac measures: ''
     Cauchy principal values: ''
     multiplication by a smooth function: ''
-    probability distribution function from a dataset: ''
```

Comment:
It looks like this is "Formule des sauts" from the original PDF, based on the items around it. Shouldn't we keep this if its listed in the syllabus?

Reply:
Or is this a totally wrong translation?

Reply:
The current line has nothing to do with the "formule des sauts". But indeed this formula could be included. I can't find an English translation. Maybe we should simply write "Derivative of the Heaviside function", which is the main content here.

Reply:
I made a general cleanup of that section which was actually very poorly translated.